docs/content/developers/release-management.md

```diff
@@ -179,3 +179,14 @@ This is done automatically by the release build on a dedicated Windows test runn
 
 !!!tip "We shouldn’t use this high-security keyfob approach to signing on the next go-around with the certs."
     It’s way too difficult to manage, and the Safenet software is atrocious.
+
+## APT and YUM/RPM Package Management
+
+The Linux `apt` and `yum/rpm` packages are built and pushed by the `nfpms` and `furies` sections of the [.goreleaser.yml](https://github.com/ddev/ddev/blob/master/.goreleaser.yml).
```

Comment:
Separate commands, add “file” to complete the thought and follow conventions we’re trying to use.

```suggestion
The Linux `apt` and `yum`/`rpm` packages are built and pushed by the `nfpms` and `furies` sections of the [.goreleaser.yml](https://github.com/ddev/ddev/blob/master/.goreleaser.yml) file.
```